plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSessionProperties.java

```diff
@@ -131,6 +132,18 @@ public HudiSessionProperties(HudiConfig hudiConfig, ParquetReaderConfig parquetR
                         IGNORE_ABSENT_PARTITIONS,
                         "Ignore absent partitions",
                         hudiConfig.isIgnoreAbsentPartitions(),
+                        false),
+                integerProperty(
+                        PARQUET_MAX_READ_BLOCK_ROW_COUNT,
```

Comment:
Please move this property above `PARQUET_SMALL_FILE_THRESHOLD` for consistency the constant order. Same for `getParquetMaxReadBlockRowCount` method.